.circleci/config.yml

```diff
@@ -495,8 +476,7 @@ steps-native-tests: &steps-native-tests
           python src/electron/script/native-tests.py run \
             --config $TESTS_CONFIG \
             --tests-dir src/out/Default \
-            --output-dir test_results \
-            --binary $(circleci tests split tests_list.txt)
```

Comment:
`unittests` execution time is heavily dominated by `net_unittests`, there are no reasons to split binaries execution, a job running `net_unittests` will take significantly more time than any other.